src/Browscap/Data/Validator/PropertiesValidator.php

```diff
@@ -0,0 +1,91 @@
+<?php
+declare(strict_types = 1);
+namespace Browscap\Data\Validator;
+
+class PropertiesValidator implements ValidatorInterface
```

Comment:
If this fully implements `ValidatorInterface`, it can be `final`

Reply:
changed